packages/x-components/src/x-modules/facets/components/lists/selected-filters-list.vue

```diff
@@ -88,6 +90,20 @@
      */
     @Prop({ default: 'ul' })
     protected animation!: Vue | string;
+
+    /**
+     * Transform a dictionary of Filters including the slot name.
+     *
+     * @returns A dictionary of facets with the slot name.
+     *
+     * @internal
+     */
+    protected mapSlot(selectedFilters: Filter[]): RenderFilter[] {
+      return selectedFilters.map((filter: Filter) => ({
+        ...filter,
+        slotName: isFacetFilter(filter) ? toKebabCase(filter['facetId'] as string) :' default'
```

Comment:
Unless you need it (kebab case properties, or dynamic properties for example), avoid using bracket syntax to access properties. It adds noise.